crates/store/re_types/definitions/rerun/archetypes/depth_image.fbs

```diff
@@ -41,17 +41,30 @@ table DepthImage (
   /// If not set, the depth image will be rendered using the Turbo colormap.
   colormap: rerun.components.Colormap ("attr.rerun.component_optional", nullable, order: 3200);
 
+  /// The expected range of depth values.
+  ///
+  /// This is typically the expected range of valid values.
+  /// Everything outside of the range is clamped to the range for the purpose of colormpaping.
+  /// Note that point clouds generated from this image will still display all points, regardless of this range.
+  ///
+  /// If not specified, the range will be automatically be determined from the data.
+  /// Note that the Viewer may try to guess a wider range than the minimum/maximum of values
+  /// in the contents of the depth image.
+  /// E.g. if all values are positive, some bigger than 1.0 and all smaller than 255.0,
+  /// the Viewer will conclude that the data likely came from an 8bit image, thus assuming a range of 0-255.
+  depth_range: rerun.components.ValueRange ("attr.rerun.component_optional", nullable, order: 3300);
+
   /// Scale the radii of the points in the point cloud generated from this image.
   ///
   /// A fill ratio of 1.0 (the default) means that each point is as big as to touch the center of its neighbor
   /// if it is at the same depth, leaving no gaps.
   /// A fill ratio of 0.5 means that each point touches the edge of its neighbor if it has the same depth.
   ///
   /// TODO(#6744): This applies only to 3D views!
-  point_fill_ratio: rerun.components.FillRatio ("attr.rerun.component_optional", nullable, order: 3300);
+  point_fill_ratio: rerun.components.FillRatio ("attr.rerun.component_optional", nullable, order: 3400);
```

Comment:
If only there existed an integer between 3200 and 3300 that `depth_range` could have used 😉

Reply:
couldn't thing of one either! damage is done, nothing to do ;-)